api/app/tests/morecast_v2/test_forecasts.py

```diff
@@ -42,6 +42,24 @@
                                            update_timestamp=end_time,
                                            update_user='test2')
 
+morecast_input_1 = MoreCastForecastInput(station_code=1,
```

Comment:
Is our existing code's use of `MorecastForecastRecord` objects wrong? It kind of seems like it might be?

Reply:
I think it's use is ok, I moved away from it in places because it's use means losing `grass curing` values, which we need for posting to WF1, but it's not needed for our DB. I might be missing something though 

If you're just referring to `test_forecasts`, these `MorecastForecastRecord` objects are still used in other tests

Reply:
Gotcha. I may make a dev task ticket to audit the use of `MoreCastForecastInput`, `MoreCastForecastOutput`, `MoreCastForecastRecord` and `WeatherIndeterminate` in our back-end. It's totally unrelated to your work, just some inconsistencies I'm noticing. For example, in `forecasts.py` the typing assigned to the `forecasts` variable in `get_forecasts()` is incorrect. The variable is typed as a `List[WeatherIndeterminate]` and this list gets returned, but the return type of the function (rightly so) is `List[MoreCastForecastOutput]`.

Reply:
I see, sounds like a good idea. For now, I could make that 1 quick change in this PR since it's clearly a typing error?

Reply:
I agree -- we could consolidate a lot of the `Morecast` structures now that we landed on a working domain model.